src/main/java/service/JudgeService.java

```diff
@@ -0,0 +1,27 @@
+package service;
+
+import domain.Car;
+import domain.Cars;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JudgeService {
+
+    public List<String> findWinners(Cars cars) {
+        List<Car> winners =  cars.getCars().stream()
```

Comment:
```suggestion
        List<Car> winners = cars.getCars().stream()
```

불필요한 띄어쓰기나 개행은 제거해주세요!